website/docs/d/pi_network.html.markdown

```diff
@@ -8,7 +8,7 @@ description: |-
 ---
 
 # ibm_pi_network
-Retrieve information about the network that your Power Systems Virtual Server instance is connected to. For more information, about power virtual server instance network, see [setting up an IBM i network install server](https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-preparing-install-server).
+Retrieve information about the network that your Power Systems Virtual Server instance is connected to. For more information, about power virtual server instance network, see [setting up an IBM network install server](https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-preparing-install-server).
```

Comment:
I feel https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-configuring-subnet this page fits here.